apps/blockchain/lib/state.ex

```diff
@@ -1,6 +1,6 @@
-defmodule CLI.State do
+defmodule Blockchain.State do
```

Comment:
what do you think about moving this module under `Blocktree` namespace. `Blockchain.Blocktree.State`

Reply:
done.